src/Integration.Vsix/Analysis/DisableRuleCommand.cs

```diff
@@ -17,15 +17,16 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-
 using System;
 using System.ComponentModel.Design;
 using System.Diagnostics;
 using System.Linq;
 using Microsoft.VisualStudio.Shell;
 using Microsoft.VisualStudio.Shell.Interop;
+using Org.BouncyCastle.Crypto.Parameters;
```

Comment:
It seems odd that the `BouncyCastle` using is required because of this change. Is it definitely required?

Reply:
It is not, I have absolutely no idea why that was put in

Reply:
Probably VS trying to be helpful when you were typing, but incorrectly guessing which types you actually needed.